jans-pycloudlib/jans/pycloudlib/persistence/sql.py

```
@@ -342,9 +590,77 @@
+
+        This method is idempotent - calling it multiple times is safe.
+        """
+        if self._closed:
+            return
+
+        self._closed = True
+
+        try:
+            _sql_client_instances.discard(self)
+        except Exception:
+            pass
```

Ruff (0.14.7)

`try`-`except`-`pass` detected, consider logging the exception

(S110)